### ISSUE TITLE
tgchat messages dissappearing

### ISSUE THREAD

**Issue by stanalbatross** [cmss13-devs/cmss13#64]
## Reproduction:

Multiple people have reported things like the ARES/command announcements sound playing but no actual message being sent, a similar bug that has been happening on tg itself since 2020? Comment in here if you have also experienced it.

**Comment by Shad0vvs** [comment]
https://github.com/tgstation/tgstation/issues/54692
Allegedly happens with goonchat too, at least on tg, def a byond issue.

**Comment by Faunenschnitt** [comment]
As I cannot use fancy chat under linux, but have the same issue, I am not sure if it is tgchat related. Happens for me most of the time when the server is under heavy load (or my client)

**Comment by SexmasterWasTaken** [comment]
Yeah it's been happening for a while now, it's not just command announcements and ARES, though. It's all messages from what I've understood and seen, you just don't notice when someone's message over radio disappears because there's no noise associated with that.

**Comment by Unknownity** [comment]
Can also occur to yourself, for example, you write out a message for hivemind but it doesn't appear for you when you send it (others can acknowledge it)

**Comment by fira** [comment]
As far as I know this is a BYOND Bug, it's been happening past decade with all chat systems across several codebases. I noticed it mainly on CM because of regular command announcements. It seems to happen more often under load (net or game? unclear)

**Comment by Huffie56** [comment]
![image](https://user-images.githubusercontent.com/117036822/237036467-d236c74b-4b7a-4458-8874-33b721bd6e3f.png)
still a thing apparently

**Comment by Wintermote** [comment]
I've never seen this happen with any server or chat implementation besides tgchat

I'm 100% certain its related to tgchat in some way. I've never seen it happen before that got implemented, not just on cm but on any server.

**Comment by Drulikar** [comment]
https://github.com/tgstation/tgstation/issues/54692#issuecomment-831824004
Has already been reported to byond

**Comment by zzzmike** [comment]
> [tgstation/tgstation#79479](https://github.com/tgstation/tgstation/pull/79479) - this might be able to be ported to help with this

https://github.com/cmss13-devs/cmss13/pull/6594 - thank you nivrak
